tests/test_users_endpoints.py

```diff
@@ -10,16 +10,16 @@
 There is NO WARRANTY, to the extent permitted by law.
 
 Description:
-    
+
 
 Resources:
 
 """
 import os
 import requests
 
-BASE = "http://127.0.0.1:4200/"
+BASE = 'http://127.0.0.1:4200/'
 endpoint = 'users'
 
-url = os.path.join(BASE, endpoint) 
-response = requests.post(url, {'email': 'contact@cannlytics.com'})
\ No newline at end of file
+url = os.path.join(BASE, endpoint)
+response = requests.post(url, {'email': 'contact@cannlytics.com'})
```

Comment:
@keeganskeate here is an example with mixed usage messing up the `sed` attempt. Do you want the following?